javascript/node/code-style-recommendations/styling-brackets.md

```diff
@@ -93,33 +105,39 @@ function wrong()
     status: 'wrong',
   };
 }
-console.log(wrong())
+console.log(wrong());
+// undefined
 ```
-Will print `undefined`. This happens because of a JavaScript property called *automatic semicolon insertion* (ASI). This property is what allows you to write JS code without having to add semicolon after each statement. Hence, the function above effectively becomes:
+
+Why is that?
+
+It turns out that JavaScript is trying to help us write correct code. 
+
+Statements in JavaScript are separated with a semicolon, i.e. `;`. If we forget to add semicolon after a line of code, JavaScript tries to be a good friend and inserts a semicolon for us. This property is called *automatic semicolon insertion* (ASI). 
```

Comment:
If we forget to add semicolon... -> If we forget to add a semicolon...

Reply:
```suggestion
Statements in JavaScript are separated with a semicolon, i.e. `;`. If we forget to add a semicolon after a line of code, JavaScript tries to be a good friend and inserts a semicolon for us. This property is called *automatic semicolon insertion* (ASI). 
```